sql_upgrade.php

```diff
@@ -18,6 +18,17 @@
 require_once('interface/globals.php');
 require_once('library/sql_upgrade_fx.php');
 
+$versionService = new \services\VersionService();
+
+$desiredVersion = new \entities\Version();
+$desiredVersion->setAcl($v_acl);
```

Comment:
Is there a way to not let this script set the acl since this is address somewhere else?